src/test/java/com/wikia/webdriver/testcases/discussions/DeleteAllPostsByUserTests.java

```diff
@@ -193,43 +197,51 @@ public void modUserDesktopDeleteAllOptionNotVisibleOnDifferentWiki() {
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   public void modUserMobileDeleteAllOptionNotVisibleOnDifferentWiki() {
     PostEntity.Data post = setUp(MercuryWikis.DISCUSSIONS_MESSAGING);
-    assertTrue(deleteAllOptionNotVisible(userWithPosts.getUserId()));
+    assertTrue(deleteAllOptionNotVisibleMobile(userWithPosts.getUserId()));
     cleanUp(post);
   }
 
   @Test(groups = "discussions-deleteAllPostsByUser")
   @Execute(asUser = User.STAFF)
   @InBrowser(browserSize = DESKTOP_RESOLUTION)
   public void modUserDesktopCanDeleteAllPostsOnModdedWiki() {
-    assertTrue(deleteAllAndReturnFirst().isDeleted());
+    assertTrue(deleteAllAndReturnFirstDesktop().isDeleted());
   }
 
   @Test(groups = "discussions-deleteAllPostsByUser")
   @Execute(asUser = User.HELPER)
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   public void modUserMobileCanDeleteAllPostsOnModdedWiki() {
-    assertTrue(deleteAllAndReturnFirst().isDeleted());
+    assertTrue(deleteAllAndReturnFirstMobile().isDeleted());
   }
 
   // HELPER METHODS
 
-  private boolean deleteAllOptionNotVisible(String userId) {
-    return getDeleteAllButton(userId).isNotVisible();
+  private boolean deleteAllOptionNotVisibleMobile(String userId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "deleteAllOptionNotVisibleMobile" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)